app/services/addCompanyRequest.js

```diff
@@ -2,10 +2,15 @@ import fetch from 'isomorphic-fetch';
 import { API_URL } from 'config';
 
 const addCompanyRequest = async data => {
+  const formData = new FormData();
+  formData.append('name', data.name);
```

Comment:
You can make this code more flexible and open to changes if you iterate through all the keys of the data object and append them to the form.